composer.json

```diff
@@ -11,7 +11,7 @@
     "minimum-stability": "dev",
     "require": {
         "php": ">=5.3.2",
-        "symfony/event-dispatcher": "^2.0 || ^3.0"
+        "symfony/event-dispatcher": "^2.3 || ^3.0"
```

Comment:
I did the check once, maybe after I was done with the PR it turns out it does run on even earlier versions of SF2.
Please note though that 2.0.x is [EOL since 09/2013](http://symfony.com/doc/current/contributing/community/releases.html#version-history). If someone still uses that, that user shouldn't expect much support for it IMHO (they sure won't get it from SF).